grunt/options.coffee

```diff
@@ -276,11 +277,14 @@ module.exports = (grunt) ->
           ]
         src: 'src/js/wrapped/uuid.core.js'
         dest: 'tmp/wrapped_uuid.js'
-
-    subgrunt:
-      bluebird:
-        projects: {}
-  #          'bower_components/bluebird': ["build","--features='core'"]
+      markerSpiderfier:
```

Comment:
Sameold wrapping but this should keep OverlappingMarkerSpiderfier local in the closure.